doc/developer/chips/10.rst

```diff
@@ -531,24 +531,76 @@ For more details on when the copy initializer would be called, please refer to
 Deinitialization
 ++++++++++++++++
 
-Records and classes can specify actions to take when a variable of that type is
-deinitialized. Record variables are deinitialized when they go out of scope and
-class variables are deinitialized when `delete` is called on them.
+Deinitialization is performed in these cases:
 
-To specify additional actions that occur upon deinitialization, specify a `proc
+ * Record variables are deinitialized when they go out of scope.
+ * Class variables are deinitialized when `delete` is called on them.
+
+To specify actions that occur upon deinitialization, user can specify a `proc
```

Comment:
This sentence is a fragment now that you've changed the second clause to no longer be in the imperative tone.  I believe this would be rectified by changing it to "the user can specify"